diff --git a/devenv-shell/src/dialect/bash.rs b/devenv-shell/src/dialect/bash.rs
--- a/devenv-shell/src/dialect/bash.rs
+++ b/devenv-shell/src/dialect/bash.rs
@@ -307,7 +307,7 @@
     }
 
     fn prompt_prefix(&self) -> &str {
-        r#"PS1="(devenv) ${PS1:-}""#
+        r#"PS1="${PS1:-}""#
     }
 
     fn format_task_exports(&self, exports: &BTreeMap<String, String>) -> String {
diff --git a/devenv-shell/src/dialect/zsh.rs b/devenv-shell/src/dialect/zsh.rs
--- a/devenv-shell/src/dialect/zsh.rs
+++ b/devenv-shell/src/dialect/zsh.rs
@@ -158,7 +158,7 @@
     }
 
     fn prompt_prefix(&self) -> &str {
-        r#"PROMPT="(devenv) ${PROMPT}""#
+        r#"PROMPT="${PROMPT}""#
     }
 
     fn format_task_exports(&self, exports: &BTreeMap<String, String>) -> String {
diff --git a/src/modules/top-level.nix b/src/modules/top-level.nix
--- a/src/modules/top-level.nix
+++ b/src/modules/top-level.nix
@@ -399,9 +399,8 @@
     enterShell = lib.mkBefore ''
       ${lib.optionalString
         (config.devenv.cli.version == null || !lib.versionAtLeast config.devenv.cli.version "2.1")
         ''
-          export PS1="\[\e[0;34m\](devenv)\[\e[0m\] ''${PS1-}"
         ''
       }
 
       # Override temp directories that stdenv set to NIX_BUILD_TOP.
